tck/src/main/java/org/eclipse/microprofile/fault/tolerance/tck/TimeoutTest.java

```diff
@@ -48,6 +53,12 @@
 
     @Deployment
     public static WebArchive deploy() {
+        final Asset config = new ConfigAnnotationAsset()
+            .setValue(TimeoutClient.class,"serviceA", Timeout.class, getConfig().getTimeoutInStr())
```

Comment:
Again, is this missing an argument?

Reply:
I see there's an argument-less variant which defaults to 1000, but I'd lean towards being explicit.